src/Lykke.Service.PayInternal/Controllers/SupervisingController.cs

```diff
@@ -0,0 +1,120 @@
+﻿using AutoMapper;
+using Common;
+using Common.Log;
+using Lykke.Common.Api.Contract.Responses;
+using Lykke.Service.PayInternal.Core.Domain.Supervisor;
+using Lykke.Service.PayInternal.Core.Services;
+using Lykke.Service.PayInternal.Extensions;
+using Lykke.Service.PayInternal.Models.Supervising;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.SwaggerGen;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Lykke.Service.PayInternal.Controllers
+{
+    [Route("api/supervising")]
+    public class SupervisingController : Controller
+    {
+        private readonly ILog _log;
+        private readonly ISupervisorService _supervisorService;
+        public SupervisingController(
+            ILog log,
+            ISupervisorService supervisorService)
+        {
+            _supervisorService = supervisorService;
+            _log = log.CreateComponentScope(nameof(SupervisingController));
+        }
+        /// <summary>
+        /// Returns list of merchants for Employee supervising
+        /// </summary>
+        /// <param name="merchantId"></param>
+        /// <param name="employeeId"></param>
+        /// <returns></returns>
+        [HttpGet]
+        [Route("{merchantId}/{employeeId}")]
+        [SwaggerOperation("GetMerchants")]
+        [ProducesResponseType(typeof(AvailableMerchantsResponseModel), (int)HttpStatusCode.OK)]
+        public async Task<IActionResult> GetMerchants(string merchantId, string employeeId)
+        {
+            if (!ModelState.IsValid)
+                return BadRequest(new ErrorResponse().AddErrors(ModelState));
+
+            try
+            {
+                var supervisor = await _supervisorService.GetAsync(merchantId, employeeId);
+                if (supervisor == null)
+                    return NotFound(ErrorResponse.Create("Couldn't find supervisor"));
+                var list = supervisor.SupervisorMerchants.Split(';').ToList();
+                return Ok(new AvailableMerchantsResponseModel { Merchants = list });
+            }
+            catch (Exception ex)
+            {
+                await _log.WriteErrorAsync(nameof(SupervisingController), nameof(GetMerchants), ex);
```

Comment:
if you use in ctor `CreateComponentScope(nameof(SupervisingController))` then sync method for logs should be used `_log.WriteError(nameof(GetMerchants), ex);`